packages/broker/src/plugins/operator/closeExpiredFlags.ts

```diff
@@ -0,0 +1,33 @@
+import { EthereumAddress, Logger } from '@streamr/utils'
+import { ContractFacade, SponsorshipResult } from './ContractFacade'
+
+const logger = new Logger(module)
+
+export const closeExpiredFlags = async (
+    flagLifetime: number,
```

Comment:
To be in sync with the `ContractFacade` method

```suggestion
    maxFlagAgeSec: number,
```